content/ADR-137-new-friend-request.md

```diff
@@ -217,19 +217,19 @@ sequenceDiagram
   participant renderer
   participant kernel
   
-renderer->kernel: GetFriendRequests(getFriendRequestsPayload{sentLimit:50, sentSkip:0, receivedLimit:50, receivedSkip:0})
+renderer->kernel: GetFriendRequestsV2(getFriendRequestsPayload{sentLimit:50, sentSkip:0, receivedLimit:50, receivedSkip:0})
```

Comment:
I think the `messageId` param is missing in `getFriendRequestsPayload`, isn't it?